docs/blog/2018-07-31-docs-redesign/index.md

```diff
@@ -0,0 +1,121 @@
+
+#### Things we liked:
+
+- Accordion menus make design clean
+- Flat menus mean nothing is hidden
+- Fast action when you click buttons
+- Task focused titles and headers
+- Explanation of why to use the software and when not to use it
+- Lots of organized "how does this work" content
+- List of “lessons”
+- Different organizational structures for different services
+
+#### Things we didn’t like:
+
+- Not enough categories
+- Cluttered landing pages
+- Conflicting menus on the same page
+- Not enough “how to get a task done” docs
+- Accordion menus make things hard to find at a glance because you can only look at one bucket at a time
+- Open menus are often unorganized
+
+As you can see, accordion menus and flat menus both have desirable pros and undesirable cons. We decided to adopt an accordion menu that will eventually be fully expandable with a toggle “open/closed” button as well as allowing you to open as many or as few buckets as you want, as opposed to one at a time. Hopefully you’ll have the best of both worlds this way!
+
+## RFC Process
+
+Because redesigning the information architecture of the .org site would affect the whole community and would take more than a day or two to accomplish, I wrote a [Docs redesign RFC](https://github.com/gatsbyjs/rfcs/pull/5) and got excellent feedback from many community members.
+
+## Card sort
+
+Many of you (36 in fact!) took the card sort to give data on how Gatsby users currently categorize existing docs. The card sort data helped inform the new structure of the docs sidebar. You can read the [card sort blogpost](https://www.gatsbyjs.org/blog/2018-06-26-card-sort-results/), which has more detail about the card sort results.
+
+## Usability testing
+
+One thing that happened (and this shouldn't surprise me) is that Gatsby community members are enthusiastic and willing to help. I included an invitation at the end of the card sort that invited card sort participants to sign up for a usability test, thinking "oh, no one will sign up for this." Then, about 15 people signed up, and I only need about 5-7 for solid data ([reference here](https://www.invisionapp.com/blog/ux-usability-research-testing/)).
+
+So, I did three different usability tests with about 5 people each and got some really valuable insights. Thank you to all these people for taking the time to meet with me for usability tests!
+
+Thank you Martyn Hoyer, Eka, Benjamin Modayil, Nicky Meuleman, Jonathan Prozzi, Kelly Lawrence, Chris Wiseman, Simon Koelewijn, Cameron Steele, Bogdan Lazar, Shannon Smith, Hugo Marques, Ria Carmin, Abhishek Vishwakarma, Peter Wiebe, and Korey Boone.
+
+Common obstacle #1:
+
+- Information embedded in the tutorial doesn't _also_ exist in the guides, like "how to create links between pages." Algolia search doesn't find keyword inside headers within a document.
```

Comment:
cc @s-pace, this is probably something misconfigured in the docsearch

Reply:
This is not true indeed.

We do match element from lvlX elements!

Reply:
ok thanks I'll edit this! You're right; I retested it and released that it was just a readability problem. I tried searching for "slugs" and the tutorial header showed up lower on the list and was so wordy, it was hard to scan. So yeah, there is no functionality problem, only a problem with how long the text is and it's basically too long for such a small box, so very hard to read quickly:

<img width="558" alt="screen shot 2018-08-01 at 10 19 00 am" src="https://user-images.githubusercontent.com/32467162/43537509-5c710a0c-9574-11e8-9470-afb3e41b7a80.png">